src/api.ts

```diff
@@ -1,5 +1,4 @@
 import request from 'superagent';
-import {API_BASE_URL} from './config';
```

Comment:
thanks for handling all this! (there are some minor things i'd like to clean up in this PR as well, just going to leave some comments as notes-to-self to address later today)